components/automate-ui/src/app/modules/infra-proxy/infra-roles/infra-roles.component.html

```diff
@@ -29,7 +29,9 @@
               <a [routerLink]="['/infrastructure','chef-servers', serverId,
               'organizations', orgId, 'roles', role.name]">{{ role.name }}</a>
             </chef-td>
-            <chef-td>{{ role.description === '' ?  '...' : role.description }}</chef-td>
+            <chef-td>
+              {{ (role.description.length &gt; 30)? (role.description | slice:0:30)+'...' : role.description }}
```

Comment:
It looks like you are trying to improve the UX by dealing with long values here, which I like. However, it is also changing the existing behavior in the case of the empty string. Is that intentional?